src/MeshTransfer.f90

```diff
@@ -416,14 +433,28 @@ SUBROUTINE setupP2P(TM,mesh_in,mesh_out)
       REAL(SRK),INTENT(IN) :: mesh_in(:)
       REAL(SRK),INTENT(IN) :: mesh_out(:)
 
+      INTEGER(SIK) :: i,j
+      REAL(SRK) :: dx
       REQUIRE(SIZE(mesh_in)>0)
       REQUIRE(SIZE(mesh_out)>0)
 
       IF(ALLOCATED(TM)) DEALLOCATE(TM)
       ALLOCATE(TM(SIZE(mesh_out),SIZE(mesh_in)))
+      TM(:,:)=ZERO
+
+      DO i=1,SIZE(mesh_out)
+        j=findIndex(mesh_in,mesh_out(i),.FALSE.,incl=1)
+        IF(j>0) THEN
+          dx=mesh_in(j+1)-mesh_in(j)
+          TM(i,j)=(mesh_in(j+1)-mesh_out(i))/dx
+          TM(i,j+1)=(mesh_out(i)-mesh_in(j))/dx
+        ELSEIF(j==-1) THEN
```

Comment:
Spacing on relational operators ```(arg1 == arg2)```

Reply:
On a side note, I've been thinking about how we can automate whitespace formatting to give the code a consistent look without needing to have all the developers remember to fix these little quirks.

Reply:
Probably more concerning is how inconsistent I was about this.  Fixed now.